src/stylesheets/elements/_buttons.scss

```diff
@@ -80,6 +80,37 @@
   background-color: transparent;
   box-shadow: none;
 
+  // Add color variants here
+  &.usa-button--secondary {
+  	@include u-border("secondary");
+    background-color: white;
+
+    &:visited {
+      color: black;
+      box-shadow: none;
+    }
+
+    &.usa-button--hover,
+    &:hover {
+      background-color: color("secondary-lighter");
```

Comment:
Just FYI, this doesn't match the mockup. But the mockup uses a non-theme (invalid) color, so we need to make Shannon/Designers aware of this issue that we had to substitute it.

Reply:
I see. I'll tell Shannon tomorrow.

Reply:
I haven't seen that mockup, but are we sure we want to color the background on the non-inverse outline button? The primary outline just uses transparent as its background. Inverse uses a colored background. Should this class really be an inverse variant?